src/main/java/uk/gov/hmcts/reform/iacasenotificationsapi/domain/handlers/presubmit/NotificationHandler.java

```diff
@@ -41,7 +41,9 @@ public NotificationHandler(BiPredicate<PreSubmitCallbackStage, Callback<AsylumCa
     public boolean canHandle(PreSubmitCallbackStage callbackStage, Callback<AsylumCase> callback) {
         requireNonNull(callbackStage, "callbackStage must not be null");
         requireNonNull(callback, "callback must not be null");
-        if (callback.getEvent() == Event.SUBMIT_APPLICATION || callback.getEvent() == Event.UPLOAD_BAIL_SUMMARY) {
+        if (callback.getEvent() == Event.SUBMIT_APPLICATION
+            || callback.getEvent() == Event.UPLOAD_BAIL_SUMMARY
+            || callback.getEvent() == Event.UPLOAD_SIGNED_DECISION_NOTICE) {
```

Comment:
@neha-aggarw, We should put all these into a list now, otherwise it'll just be a lot of ORs soon. Something like this:
![image](https://user-images.githubusercontent.com/99262639/168247112-f13931ff-87b7-47e4-8d35-3b715c5b13e7.png)

Reply:
I've added this now (including the third event)